foundation/foundation-mda/src/main/resources/templates/data-delivery-spark/pipeline.pom.xml.vm

```diff
@@ -128,6 +128,10 @@
             <groupId>com.boozallen.aissemble</groupId>
             <artifactId>foundation-metadata-producer</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.boozallen.aissemble</groupId>
+            <artifactId>extensions-messaging-kafka</artifactId>
+        </dependency>
```

Comment:
Q: is this a default inclusion for all spark pipeline?

Reply:
Yes. Unfortunately all our pipelines include:
```
<dependency>
    <groupId>io.smallrye.reactive</groupId>
    <artifactId>smallrye-reactive-messaging-in-memory</artifactId>
    <version>${version.smallrye.reactive.messaging}</version>
    <scope>test</scope>
</dependency>
```
by default. So in order to remain compatible with that, we must include the new `extensions-messaging-kafka` by default.